pkg/agent/flowexporter/types.go

```diff
@@ -86,6 +86,7 @@ type Connection struct {
 	EgressIP                             string
 	AppProtocolName                      string
 	HttpVals                             string
+	EgressNode                           string
```

Comment:
If you are not going to add this information to the exported IPFIX record, then I don't think the FlowExporter changes in this PR make sense.
And if you do want to add the information to the record, it should probably be done in a separate PR (a change to go-ipfix is also required), given that this one is specific to Traceflow.

Reply:
reverted, thanks.